packages/core/src/styles/components/_toolbar.scss

```diff
@@ -0,0 +1,80 @@
+$css--reset: false;
+@import "./../vendor/carbon-components/src/components/overflow-menu/overflow-menu";
+
+g.#{$prefix}--#{$charts-prefix}--toolbar {
+	@include carbon--motion(standard, expressive);
+
+	.toolbar-container {
+		.toolbar-button {
+			fill: $ui-background;
+			cursor: pointer;
+			&:hover {
+				.toolbar-button-background {
+					fill: $hover-ui;
+				}
+				.toolbar-button-background--disabled {
+					fill: $ui-background;
+				}
+			}
+
+			&--focusable {
+				&:focus {
+					.toolbar-button-background {
+						stroke: $icon-02;
+						stroke-width: 2;
+						stroke-dasharray: 2;
+					}
+				}
+			}
+
+			&-icon {
+				pointer-events: visible;
+				circle {
+					fill: $icon-02;
+				}
+				rect {
+					fill: $icon-02;
+				}
+				path {
+					fill: $icon-02;
+				}
+				polygon {
+					fill: $icon-02;
+				}
```

Comment:
```suggestion
				circle, rect, path, polygon {
					fill: $icon-02;
				}
```

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/791/commits/56b3c20d8ac2990a06938ac3cd80d0d54c8bde20.